### DISCUSSION TITLE
Pi Zero W with Elite Rizer

### DISCUSSION THREAD

**Discussion by smulrine** [cagnulein/qdomyos-zwift#737] [Q&A]
Apparently I have to start qdomyos-zwift with the -qml flag to allow an Elite Rizer to connect to it, but it always just exits, unless I also use -no-gui in which case my Neo connects fine but the Rizer doesn't react at all. I have tried all the possible options for QT_QPA_PLATFORM; eglfs works but not when I use the -qml flag. Installed the Pixel desktop so I could try xcb but it didn't help. (I notice from strace that qz is trying to access files named for Roberto Viola.) Is there something I am missing, or some other flag I should be using?

**Comment by cagnulein** [comment]
Did you install all the qml modules with apt?

**Reply by smulrine** [reply]
I did not, so that will certainly have something to do with it. First time I've come across qml at all. It looks like there are a load of modules - is there an easy way to do it?

**Reply by cagnulein** [reply]
you can use wildcards to install all of them
something like apt install qml* or similar
